pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/fullalloydbcluster/_generated_object_fullalloydbcluster.golden.yaml

```diff
@@ -62,8 +61,6 @@ spec:
   networkConfig:
     networkRef:
       name: computenetwork-${uniqueId}
-  networkRef:
```

Comment:
Is this a dup field with the networkConfig.networkRef? I'm wondering why the golden object is different (I was expecting the presubmit check to block the original change).

Reply:
Good question, I don't know.  Maybe @anhdle-sso knows (is this what you meant by sharing the PR review?)

The presubmits weren't running here before, because we didn't have a mock for KMSCryptoKey